.travis/maybe-bump-version.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 
-if [ "$TRAVIS_NODE_VERSION" != "8.1.2" ]
+if [[ ! "${PUBLISH_NODE_VERSION:-8.1.2}" =~ ^$TRAVIS_NODE_VERSION ]]
 then
-  echo "Skipping version bump for TRAVIS_NODE_VERSION ${TRAVIS_NODE_VERSION}"
+  echo "Skipping pr-bumper for TRAVIS_NODE_VERSION [${TRAVIS_NODE_VERSION}]"
   exit 0
 fi
 
 if [ "$EMBER_TRY_SCENARIO" != "ember-default" ]
 then
-  echo "Skipping version bump for EMBER_TRY_SCENARIO ${EMBER_TRY_SCENARIO}"
+  echo "Skipping pr-bumper for EMBER_TRY_SCENARIO [${EMBER_TRY_SCENARIO}]"
```

Comment:
This line read differently than: https://github.com/ciena-frost/ember-frost-core/pull/546/files#diff-b29511921f39def9f1eec87ca24e0e4dR11

Reply:
This file doesn't align with https://github.com/ciena-blueplanet/pr-bumper/blob/master/.travis/maybe-bump-version.sh either - going to have to revisit that